### PR TITLE
document location__country in events SQL view

### PR DESCRIPTION
**Pull request by jaredmcdonald**

### CODE REVIEW

File: sql_data_dictionary/README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ The `events` view includes events owned by the Organization and events owned by
 | location__address_line_2 | varchar(250) | The second line of the address |
 | location__locality | varchar(250) | The city |
 | location__region | varchar(2) | The two-character state code |
```
Comment (ramillim) [inline sql_data_dictionary/README.md]: We might want to elaborate that this only includes the 50 states + DC now so as not to be confused with the territories in `location__country`. DC is technically a territory.

File: sql_data_dictionary/README.md [hunk 2]
```diff
@@ -55,6 +55,7 @@ The `events` view includes events owned by the Organization and events owned by
...
+| location__country | varchar(2) | The [ISO-3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code; note that U.S. territories have distinct country codes, e.g., Puerto Rico is `PR` |
```
Comment (ramillim) [inline sql_data_dictionary/README.md]: We could say "U.S. commonwealths and territories" to be precise.

File: sql_data_dictionary/README.md (hunk 1 above)
Comment (jaredmcdonald) [reply sql_data_dictionary/README.md]: done

File: sql_data_dictionary/README.md (hunk 2 above)
Comment (jaredmcdonald) [reply sql_data_dictionary/README.md]: done

**Pull request merged**